src/android/src/main/java/io/radar/cordova/RadarCordovaPlugin.java

```diff
@@ -87,6 +89,8 @@ public boolean execute(String action, final JSONArray args, final CallbackContex
                 offError(args, callbackContext);
             } else if (action.equals("startTrip")) {
                 startTrip(args, callbackContext);
+            } else if (action.equals("updateTrip")) {
```

Comment:
i think we should expose getTripOptions + make the status value optional.

Reply:
I can add this if its blocking / needed. It looks like `status` is optional below?